worker/runner.ts

```diff
@@ -79,6 +97,16 @@ export class AgentRunner {
       this.logger.error(`Failed to start agent server: ${error}`);
     }
 
+    try {
+      await this.browserManager?.initialize();
```

Comment:
This should be the only line we need in this file. The browserApp express server should happen within the BrowserManager